engines/python/setup/djl_python/deepspeed.py

```diff
@@ -368,7 +368,7 @@ def inference(self, inputs: Input):
             outputs.add_property("content-type", "application/json")
         except Exception as e:
             logging.exception("DeepSpeed inference failed")
-            outputs = Output().error((str(e)))
+            raise e
```

Comment:
This will kill the python process and cause SM restart the host, we don't really know if this error is recoverable or not